### ISSUE TITLE
Quarry try to destory water flow

### ISSUE THREAD

**Issue by Azuxul** [BuildCraft/BuildCraft#3958]
**BuildCraft** version: 7.99.13
**Forge** version: 14.23.1.2565
Singleplayer or multiplayer: multiplayer
Additional information:

Hi,

My quarry try to destroy water flow so this stop quary. I think quarry must ignore water and water flow or directy break water block.

I don't know if is realy a bug or if is a feature but i think is a bug.

Screenshot : 
![2018-01-01_13 08 23](https://user-images.githubusercontent.com/11092555/34467561-db67029c-eef5-11e7-8e05-380f08933765.png)

**Comment by AEnterprise** [comment]
yes it should ignore flowing water, the quarry already does this when digging but apearently not for clearing the area, for now just get rid of the source blocks yourself and it'll finish setting up to start digging
